marketplace/wpp_templates/tasks.py

```diff
@@ -14,6 +14,26 @@
 )
 
 
+def delete_unexistent_translations(app, templates):
+    templates_message = app.template.all()
+    templates_ids = [item["id"] for item in templates["data"]]
+
+    for template in templates_message:
+        template_translation = TemplateTranslation.objects.filter(
+            template=template
+        )
+        for translation in template_translation:
+            if translation.message_template_id not in templates_ids:
+                print(
```

Comment:
Does the message really need to be displayed in the terminal?  

If you need to display the messages you can use the `"log"` for example:
`log.info, log.warning, log.exception `...
 some of these examples , are visible in the sentry if necessary.